### ISSUE TITLE
Baseline models do not support Multivariate Data

### ISSUE THREAD

**Issue by abdulfatir** [awslabs/gluon-ts#2689]
## Description
Several baseline models (e.g., trivial predictors, seasonal naive) do not support multivariate data.

## Environment
- Operating system: MacOS 12.6
- Python version: 3.8
- GluonTS version: 0.12.3
- MXNet version:

(Add as much information about your environment as possible, e.g. dependencies versions.)

**Comment by jaheba** [comment]
I don't think this is a "bug". Rather we should enhance these models to also support multivariate time series.
